hack/vendor.sh

```diff
@@ -55,9 +55,39 @@ clone hg code.google.com/p/go.net 84a4013f96e0
 clone hg code.google.com/p/gosqlite 74691fb6f837
 
 #get libnetwork packages
-clone git github.com/docker/libnetwork 005bc475ee49a36ef2ad9c112d1b5ccdaba277d4
+clone git github.com/docker/libnetwork a09a60a123b4106954e5b42ed5ec41f264eeb8bf
 clone git github.com/vishvananda/netns 008d17ae001344769b031375bdb38a86219154c6
 clone git github.com/vishvananda/netlink 8eb64238879fed52fd51c5b30ad20b928fb4c36c
+clone git github.com/BurntSushi/toml f706d00e3de6abe700c994cdd545a1a4915af060
+clone git github.com/deckarep/golang-set ef32fa3046d9f249d399f98ebaf9be944430fd1d
+
+#libnetwork uses swarm discovery and store libraries
+clone git github.com/docker/swarm 54dfabd2521314de1c5b036f6c609efbe09df4ea
```

Comment:
@LK4D4 @tianon if we are okay with this approach, then yes, having a generic function to filter out unused packages & pulling in only the necessary packages will be helpful.